### ISSUE TITLE
Mumble fails to build

### ISSUE THREAD

**Issue by nyanloutre** [NixOS/nixpkgs#60013]
## Issue description

Hydra log:
https://hydra.nixos.org/build/92486838
```
In file included from BanEditor.cpp:36:0:
Global.h:142:11: error: expected ',' or '...' before '(' token
 #define g (*Global::g_global_struct)
           ^
compilation terminated due to -Wfatal-errors.
```

This might be related to this upstream issue:
https://github.com/mumble-voip/mumble/issues/3617

I tried to apply the PR fixing the above issue but the patch is on master and therefore can't be applied directly to 1.2.19 without conflicts

### Steps to reproduce

`nix-build -A mumble` on latest nixpkgs master

**Comment by nyanloutre** [comment]
A quick fix would be to use protobuf 3.6 instead of 3.7

**Comment by worldofpeace** [comment]
> A quick fix would be to use protobuf 3.6 instead of 3.7

That would be acceptable.

**Comment by nyanloutre** [comment]
I opened a PR : #60062
